tools/autograd/gen_python_functions.py

```diff
@@ -475,7 +477,7 @@ def emit_single_dispatch(declaration, is_python_method, output_gap=0):
 
     for i, arg in enumerate(args):
         unpack = is_scatter(arg) or (has_options and is_tensor_self(arg))
-        arg_expr, unpack_stmts = parse_arg(arg, i, unpack_to_local=unpack)
+        arg_expr, unpack_stmts = parse_arg(declaration['name'], arg, i, unpack_to_local=unpack)
```

Comment:
super duper OCD micro nit, could hoist `declaration['name']`

...and yes I'm really only thinking about how much shorter the line would get ugh